tests/CSharp/CSharp.h

```diff
@@ -1217,3 +1217,13 @@ void* TestIndexedProperties::operator[](size_t n) const
 extern const ComplexArrayElement ArrayOfVariableSize[];
 
 void useStdStringJustAsParameter(std::string s);
+
+typedef int (typedefedFuncPtr)(Foo *a, Bar b);
+
+class DLL_API HasFuncWithTypedefedFuncPtrAsParam
```

Comment:
As I said in the very beginning, we don't need an entire class for this. Use a single function.